Navigat1on/Date/ArrayPhotos.swift

```diff
@@ -6,7 +6,13 @@
 //
 
 import Foundation
+import UIKit
+
+//class Photos {
+//    static var photos: [String] = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20"]
+//}
+
 
 class Photos {
-    static var photos: [String] = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20"]
+    static var photos: [UIImage?] = [UIImage(named: "1"), UIImage(named: "2"), UIImage(named: "3"), UIImage(named: "4"), UIImage(named: "5"), UIImage(named: "6"), UIImage(named: "7"), UIImage(named: "8"), UIImage(named: "9"), UIImage(named: "10"), UIImage(named: "11"), UIImage(named: "12"), UIImage(named: "13"), UIImage(named: "14"), UIImage(named: "15"), UIImage(named: "16"), UIImage(named: "17"), UIImage(named: "18"), UIImage(named: "19"), UIImage(named: "20")]
```

Comment:
Гораздо быстрее можно создать:
```suggestion
    static var photos: [UIImage?] = Array(1...20).map { UIImage(named: String($0)) }
```